api/src/main/java/org/apache/iceberg/Table.java

```diff
@@ -221,4 +221,11 @@ default AppendFiles newFastAppend() {
    * @return a {@link LocationProvider} to provide locations for new data files
    */
   LocationProvider locationProvider();
+
+  /**
+   * Create a new {@link ExpireTableMetadata expire API} to manage table metadata files in this table and commit.
+   *
+   * @return a new {@link ExpireTableMetadata}
+   */
+  ExpireTableMetadata expireTableMetadata();
```

Comment:
Should this be done as part of `expireSnapshots`? I'd like to avoid adding a lot of operations to `Table` because it is already a large API.